src/problem2.rb

```diff
@@ -0,0 +1,18 @@
+class Problem2
+  def identify_largest_numbers(array)
+    raise "Array of length too small, min 2!" if array.length < 2
+
+    max1, max2 = -1
```

Comment:
This is assuming that there are no negative numbers in the array, `integers` can have negative values.